src/dotnet/CarbonAware.WebApi/api.yaml

```diff
@@ -5,16 +5,67 @@ info:
   title: Carbon Aware REST API
 
 paths:
+  '/emissions/forecasts/current':
+    get:
+      summary: Get the current forecasted marginal carbon intensity for a given set of locations.
+      description: Gets the most recent marginal carbon intensity forecast and returns the optimal start time for a window of a given duration.
+      operationId: getCurrentMarginalCarbonIntensityForecast
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
Would it possible to have an id for the `EmissionData` instance, given the fact that there are going to be multiple of these and it would be good to know which one is the optimal based on the list.

Reply:
This array is of forecasts for a list of locations.  Each forecast does pull out the optimal data point in addition to listing the entire forecast.

https://github.com/microsoft/carbon-aware-sdk/pull/63/files#diff-64ed0f8eae19da47af06ab2f84981c9638ed8a6351b49fded82561872eb7c06cR131-R132

Is that what you mean, or something else?

Reply:
Oops sorry, it is the other array (line 142)

Reply:
Right, gotcha.  Yea, I had the same thought.  Rather than figuring out an ID for something that doesn't have a natural ID already I thought it would be easier/friendlier to just include that object outright.  It's pulled out in `optimalDataPoint` and also exists in the full dataset.

```
{
          "generatedAt": "2022-05-23T00:00:00Z",
          "optimalDataPoint": {
            "location": "eastus",
            "time": "2022-05-23T00:10:00Z",
            "rating": 173.3
          },
          "location": "eastus",
          "notBefore": "2022-05-23T00:00:00Z",
          "notAfter": "2022-05-23T00:21:00Z",
          "forecastData": [
            {
              "location": "eastus",
              "time": "2022-05-23T00:05:00Z",
              "rating": 195.2
            },
            {
              "location": "eastus",
              "time": "2022-05-23T00:10:00Z",
              "rating": 173.3
            },
            {
              "location": "eastus",
              "time": "2022-05-23T00:15:00Z",
              "rating": 222.4
            },
            {
              "location": "eastus",
              "time": "2022-05-23T00:20:00Z",
              "rating": 200.1
            },
          ]
```

Reply:
I was thinking if it is there is a property that can be take it for 'free' from the request that can be used. But if not, I agree to keep this way.